src/main/java/seedu/address/MainApp.java

```diff
@@ -143,21 +146,22 @@ protected Config initConfig(Path configFilePath) {
      */
     protected UserPrefs initPrefs(UserPrefsStorage storage) {
         Path prefsFilePath = storage.getUserPrefsFilePath();
-        logger.info("Using prefs file : " + prefsFilePath);
+        logger.info("Using preference file : " + prefsFilePath);
 
         UserPrefs initializedPrefs;
         try {
             Optional<UserPrefs> prefsOptional = storage.readUserPrefs();
             if (!prefsOptional.isPresent()) {
-                logger.info("Creating new prefs file.");
+                logger.info("Creating new preference file " + prefsFilePath);
             }
             initializedPrefs = prefsOptional.orElse(new UserPrefs());
         } catch (DataConversionException e) {
-            logger.warning("UserPrefs file at " + prefsFilePath + " is not in the correct format. "
-                    + "Using default user prefs");
+            logger.warning("Preference file at " + prefsFilePath
+                    + " might be missing or corrupted. Using default preferences.");
```

Comment:
Any reason for the change of the log message text here?

Reply:
In the previous PR, #152, the "Prefs file" is used to describe the preference file. Thus I wanted to standardize it to the same term and replaced UserPrefs with Preference instead.

Reply:
Sorry, I should have been more specific. Why go from 'not in the correct format' to 'might be missing or corrupted'?

Reply:
I have standardized all corrupted warnings to use " is not in the correct format."

Reply:
In this case it is more about giving the accurate error message rather than standardizing. Which one is more accurate?

Reply:
Not in the correct format/corrupted will be more accurate here. Since missing file will be reflected by "Creating XXX file." log

Reply:
I see. I guess we can keep it as 'not in the correct format' (the incorrect format may be caused by many reasons, including file corruption).